pkg/auth/providers/providers.go

```diff
@@ -27,15 +27,15 @@ var (
 	ProviderNames          = make(map[string]bool)
 	ProvidersWithSecrets   = make(map[string]bool)
 	UnrefreshableProviders = make(map[string]bool)
-	providers              = make(map[string]common.AuthProvider)
+	Providers              = make(map[string]common.AuthProvider)
```

Comment:
Why does this need to be exported?

Reply:
The tests need to set it https://github.com/rancher/rancher/pull/38894/commits/5ebecc2cb448506abaea0e6055cc6692b272650e#diff-669858130ce986f4a18774a2b9b922d11d6cd64d1e8be3c308d8057e007ec0c7R283

We can't just call providers.Configure because there's no ScaledContext in the unit tests.